### ISSUE TITLE
Home IDP extension doesn't work as intended to

### ISSUE THREAD

**Issue by SangI762** [p2-inc/keycloak-orgs#124]
Hello, I saw that you modified a bit behaviour with Home IDP extension and using 
`"home.idp.discovery.org": "org-id"` to get specific organization idp, but I'm trying to use default behaviour with `"home.idp.discovery.domains": "domain-name",` and it doesn't work. 
Am I missing something? Is it possible to use both methods?

**Comment by xgp** [comment]
No. We override the IdP lookup behavior. You can install the original extension if you need both mechanisms.

**Comment by SangI762** [comment]
Ok, thanks. Will `"home.idp.discovery.org"` work with original extension too?

**Comment by SangI762** [comment]
Well, that would be awesome if you will keep original behaviour by domains when home.idp.discovery.org property is missing.

**Comment by xgp** [comment]
Thanks. Looking forward to your PR.
